dico_toolbox/graph.py

```diff
@@ -62,27 +92,135 @@ def stack_buckets(vertex, labels=["aims_ss", "aims_bottom", "aims_other"]):
     except:
         stack = None
         print(vertex_bcks)
-    
+
     return stack
 
 
-def get_bucket_from_graph_by_suclus_name(graph, sulcus_name, ICBM2009c, labels=["aims_ss", "aims_bottom", "aims_other"]):
-    """
-    Get and stack all buckets in the graph corresponding to sulcus name and labels.
-    
-    if ICBM2009c is True, the buckets are transformed into the ICBM2009c Template coordinates.
+def list_buckets(graph, key=None, needed=None, return_keys=None, defaults=None, transform=None, bck_types=BUCKETS_TYPES):
+    """ Stack all buckets of the graph that correspond to needed key values and bucket types.
+        Also return values of each vertex for the specified keys.
+
+        Parameters
+        ==========
+        graph: soma.aims.Graph | str
+            AIMS Cortical Graph
+
+        key: str (opt.)
+            Select vertices that have key property.
+
+        needed: list | single value (opt.)
+            Select vertices that have key value equal to one of the needed values.
+
+        return_keys: list | str | None (opt.)
```

Comment:
je n'ai pas bien compris ça. Je propose quelque chose comme çá:

Toghether with the buckets, this function can extract other properties from each grap vertex.
The properties to extract can be indicated in this parameter as a string or a list of str.
- If a string is given, the function returns a list containing the property values
- if a list of string is used, the return is a dictionnary of values corresponding to each given key.